packages/engine/Source/Scene/Model/ModelSceneGraph.js

```diff
@@ -471,13 +546,14 @@ ModelSceneGraph.prototype.buildDrawCommands = function (frameState) {
     modelPipelineStage.process(modelRenderResources, model, frameState);
   }
 
-  const modelPositionMin = Cartesian3.fromElements(
+  // Positions are in local glTF scene coordinates
```

Comment:
Is this a terminology change, is there a difference between model coordinates and local glTF scene coordinates?